cmake/SSGCommon.cmake

```diff
@@ -888,9 +888,25 @@ macro(ssg_build_html_stig_tables PRODUCT STIG_PROFILE DISA_STIG_VERSION)
         DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/transforms/xccdf2table-profileccirefs.xslt"
         COMMENT "[${PRODUCT}-tables] generating HTML STIG test info document"
     )
+    add_custom_command(
+        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/unlinked-stig-${PRODUCT}-xccdf.xml"
```

Comment:
no need for ${PRODUCT} in the name, it's in CURRENT_BINARY_DIR so it will be in build/RHEL/7